xonsh/ptk/shortcuts.py

```diff
@@ -0,0 +1,451 @@
+"""A prompt-toolkit inspired shortcut collection."""
```

Comment:
A whole bunch of PEP8 stuff here, though maybe we don't care if this is mostly from outside xonsh (from forking prompt_toolkit).